<!DOCTYPE html>
<html lang="en">
    <head>
        <title>displaCy</title>
    </head>

    <body style="font-size: 16px; font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'; padding: 4rem 2rem; direction: ltr">
<figure style="margin-bottom: 6rem">
<div class="entities" style="line-height: 2.5; direction: ltr"></br> ==== Front</br> EBioMedicineEBioMedicineEBioMedicine2352-3964Elsevier S2352-3964(18)30123-310.1016/j.ebiom.2018.04.002Research PaperA Systems Approach to Refine Disease Taxonomy by Integrating Phenotypic and Molecular Networks Zhou Xuezhong a1Lei Lei b1Liu Jun c1Halu Arda d1Zhang Yingying cfLi Bing bcGuo Zhili cgLiu Guangming aSun Changkai hijkLoscalzo Joseph eSharma Amitabh amitabh.sharma@channing.harvard.edude⁎Wang Zhong wangzh@mail.cintcm.ac.cnc⁎⁎a School of Computer and Information Technology and Beijing Key Lab of Traffic Data Analysis and Mining, Beijing Jiaotong University, No.3 Shangyuancun, Haidian District, Beijing 100044, Chinab Institute of Information on Traditional Chinese Medicine, China Academy of Chinese Medical Sciences, No.16, Nanxiaojie, Dongzhimennei, Dongcheng District, Beijing 100700, Chinac Institute of Basic Research in Clinical Medicine, China Academy of Chinese Medical Sciences, No.16, Nanxiaojie, Dongzhimennei, Dongcheng District, Beijing 100700, Chinad Channing Division of Network Medicine, Department of Medicine, Brigham and Women's Hospital, Harvard Medical School, 181 Longwood Avenue, Boston, MA 02115, USAe Department of Medicine, Brigham and Women's Hospital, Harvard Medical School, 75 Francis Street, Boston, MA 02115, USAf Dongzhimen Hospital, Beijing University of Chinese Medicine, No.5 Haiyuncang, Dongcheng District, Beijing 100700,Chinag Jiaxing Traditional Chinese Medicine Affiliated Hospital of Zhejiang Chinese Medical University, No. 1501, Zhongshan East Road, Jiaxing, Zhejiang 314000, Chinah School of Biomedical Engineering, Faculty of Electronic Information and Electrical Engineering, Dalian University of Technology, Dalian, 116024, Chinai Research Center for the Control Engineering of Translational Precision Medicine, Dalian University of Technology, Dalian 116024, Chinaj State Key Laboratory of Fine Chemicals, Dalian R&amp;D Center for Stem Cell and Tissue Engineering, Dalian University of Technology, Dalian 116024, Chinak Liaoning Provincial Key Laboratory of Cerebral Diseases, Institute for Brain Disorders, Dalian Medical University, Dalian 116044, China⁎ Correspondence to: A. Sharma, Channing Division of Network Medicine, Department of Medicine, Brigham and Women's Hospital, Harvard Medical School, 181 Longwood Avenue, Boston, MA 02115, USA. amitabh.sharma@channing.harvard.edu⁎⁎ Correspondence to: Z. Wang, Institute of Basic Research in Clinical Medicine, China Academy of Chinese Medical Sciences, No.16, Nanxiaojie, Dongzhimennei, Dongcheng District, Beijing 100700, China. wangzh@mail.cintcm.ac.cn1 These authors contributed equally to this work.</br> </br> 06 4 2018 5 2018 06 4 2018 31 79 91 27 11 2017 14 3 2018 3 4 2018 © 2018 The 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    Authors2018This
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 is an open access article under the CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).The International Classification of Diseases (ICD) relies on clinical features and lags behind the current understanding of the molecular specificity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pathobiology, necessitating approaches that incorporate growing biomedical data for classifying 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 to meet the needs of precision medicine. Our analysis revealed that the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    heterogeneous
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 molecular diversity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 chapters and the blurred boundary between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories in ICD should be further investigated. Here, we propose a new classification of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (NCD) by developing an algorithm that predicts the additional categories of a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 by integrating multiple networks consisting of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 phenotypes and their molecular profiles. With statistical validations from phenotype-genotype associations and interactome networks, we demonstrate that NCD improves 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 specificity owing to its overlapping categories and polyhierarchical structure. Furthermore, NCD captures the molecular diversity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 and defines clearer boundaries in terms of both phenotypic similarity and molecular associations, establishing a rational strategy to reform 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy.</br> </br> Highlights</br> • The International Classification of Diseases (ICD) lags behind the current molecular characteristics of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
.</br> </br> • We quantified the limitations (specificity and blurred boundary) of ICD with integrated phenotypic and molecular profiles.</br> </br> • An integrative 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network integrating phenotypic and genotypic profiles proposes a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    refined disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 category framework.</br> </br> </br> </br> Disease taxonomy is one of the foundations of medical science and healthcare solutions. The most widely used 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy in clinical settings is the International Classification of Diseases (ICD), a system established &gt;100 years ago and maintained by the World Health Organization to track 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 incidence. It is well recognized that ICD, which is based on clinical observations, largely lags behind the molecular achievements of this medical big data era. We quantified the limitations of ICD using integrated phenotypic and molecular profiles and proposed a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    refined disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy with possible applications for precision medicine.</br> </br> Keywords</br> Disease taxonomyNetwork medicineDisease phenotypesMolecular profilesPrecision medicine</br> ==== Body</br> 1 Introduction</br> Disease taxonomy plays an important role in defining the diagnosis, treatment, and mechanisms of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    human diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
. The principle of the current 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    clinical disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomies, in particular the International Classification of Diseases (ICD), goes back to the work of William Farr in the nineteenth century and is primarily derived from the differentiation of clinical features (e.g. symptoms and micro-examination of diseased tissues and cells) (Council et al., 2011). Despite its extensive clinical use, this classification system lacks the depth required for precision medicine with the limitations of its rigid hierarchical structure and, moreover, it does not exploit the rapidly expanding molecular insights of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 phenotypes. For example, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    many diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (e.g. 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    cancer
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    chronic inflammatory diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
) in the current 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomies have either high genetic heterogeneity (Bianchini et al., 2016; McClellan and King, 2010) or manifestation diversity (Arostegui et al., 2014; Jeste and Geschwind, 2014; Mannino, 2002), which give 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    little basis
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 for tailoring treatment to a patient's pathophysiology. Furthermore, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 comorbidities (Hu et al., 2016; Lee et al., 2008; Hidalgo et al., 2009), temporal 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 trajectories (Jensen et al., 2014) in clinical populations, various molecular relationships between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-associated cellular components and their connections in the interactome (Blair et al., 2013; Goh et al., 2007; Barabasi et al., 2011; Rzhetsky et al., 2007; Zhou et al., 2014), and many successful drug repurposing cases (Li and Jones, 2012; Chong and Sullivan Jr., 2007; Ashburn and Thor, 2004; Wu et al., 2016; Evans et al., 2005) altogether demonstrate the vague boundary between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    different diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in current 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomies. Moreover, the deep understanding of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 based on the advances in 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 biology, bioinformatics, and multi-omics data necessitates the reclassification of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy (Mirnezami et al., 2012).</br> </br> In the past decade, efforts to reclassify 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 based on molecular insights have increased with studies related to molecular-
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    based disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 subtyping in different 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 conditions, such as 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    acute leukemias
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (Golub et al., 1999; Alizadeh et al., 2000), 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    colorectal cancer
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (Dienstmann et al., 2017), 
<mark class="entity" style="background: #FF0000; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    oesophageal carcinoma
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FN</span>
</mark>
 (Cancer Genome Atlas Research et al., 2017), 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    pancreatic cancer
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (Bailey et al., 2016), 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    cancer metastasis
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (Chuang et al., 2007), 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    neurodegenerative disorders
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (Mann et al., 2000), 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    autoimmunity disorders
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (Ahmad et al., 2003), multiple 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    cancer
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 types across tissues of origin (Hoadley et al., 2014), and a network-based stratification method for 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    cancer
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 subtyping (Hofree et al., 2013). Further insights will arise from integrating all types of biomedical data with a single framework to exploit 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease-disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 relationships. Data integration methods that utilize multiple types of data, including ontological and omics data, have been used to classify and refine 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 relationships (Gligorijevic and Przulj, 2015; Menche et al., 2015; Gligorijevic et al., 2016). Despite these efforts, the development of a molecular-based 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy that links molecular networks and pathophenotypes still remains challenging (Menche et al., 2015; Hofmann-Apitius et al., 2015; Jameson and Longo, 2015).</br> </br> Here, we aim to refine a widely used 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    clinical disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 classification scheme, the ICD. To achieve this, we first quantify the category similarity among the ICD chapters using ontology-based similarity measures and investigate the molecular connections of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs in the same ICD chapters. Furthermore, we seek the correlation between category and molecular similarity, and check for the heterogeneity of molecular specificity and correlated boundary between categories in ICD taxonomy. Finally, we construct a new classification of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (NCD) with overlapping structures. The aim is to provide clear boundaries between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    distinct diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 belonging to different categories using a new 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 classification scheme (Fig. 1 &amp; Fig. S3).Fig. 1 Overview of the new 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy construction and validation. a. Similarity calculation between the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs in ICD taxonomy, including the calculation of 1) category similarity; 2) Phenotype similarity (based on ICD-MeSH term mapping) and 3) Molecular profile similarities (based on ICD-UMLS term mapping) of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs in ICD; b. Module or community annotations of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 association network by chapters in ICD or NCD. We generate 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 association network, in which nodes represent 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 and the link weights represent their corresponding phenotype or molecule profile similarities. The module annotations of the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network correspond to ICD chapters or NCD categories; c. Construction of integrated 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network (IDN) and generation of NCD. The links of IDN are fused from the multiple similarities (e.g. phenotype similarity and shared gene similarity). Based on IDN, NCD is generated by community detection algorithms with overlapping 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 members; d. Quality evaluation and validation of ICD and NCD. The molecular specificity (or inverse molecular diversity) and network modularity are used for evaluation and comparison of the quality of two 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomies. Furthermore, we validate the robustness of NCD with two independent phenotype-genotype association datasets, namely GWAS and PheWAS.</br> </br> Fig. 1</br> </br> 2 Materials and Methods</br> 2.1 Basic Dataset Compilation</br> In this work, large curation efforts are performed to generate the related data sources (details see Supplementary Materials (
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
) Section 1). We obtained the updated text version of ICD-9-CM (2011) and extracted the list of ICD codes with their hierarchical structures. While we recognize the improvements of the currently used ICD-10 over ICD-9, nevertheless, we chose to use ICD-9-CM as the adoption of ICD-10 has been slow in the United States (Butler, 2014) and since it was still being widely used at the time of the data collection for this paper (Blair et al., 2013; Wang et al., 2017). Furthermore, although ICD-10 does have more codes than ICD-9-CM, the structure is kept almost the same. We obtained the high-quality phenotype-genotype (
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-gene) associations from Disease Connect database (2015 version) (Liu et al., 2014), leaving out the less reliable text mining entries and focusing only on Genome-wide association study (GWAS), Online Mendelian Inheritance in Man (OMIM) and differential expression evidence types, and manually mapped those 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in unified medical language system (UMLS) codes to ICD and MeSH codes (
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 1.6).</br> </br> To calculate the molecular network and phenotype characteristics related to 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 phenotypes, a high-quality subset of human protein-protein interactions was filtered from STRING V9.1 (Franceschini et al., 2013) using the score threshold at ≥ 700, as well as a well-
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    established disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-phenotype (
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-symptom) association dataset (i.e. 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network with symptom similarity, HSDN) (Zhou et al., 2014) derived from PubMed bibliographic records and the gene ontology annotations from NCBI gene database are adopted. To ensure the results are not biased by computational predictions in the STRING database, we replicated the classification pipeline with manually curated PPI networks (Menche et al., 2015), which rely only on physical protein interactions with experimental support, and found that the results are robust (
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 8.3).</br> </br> In addition, to validate the robustness of our results from independent data sources, we filtered the GWAS and Phenome Wide Association Studies (PheWAS) data from University of California Santa Cruz (UCSC) Genome Browser (Tyner et al., 2017) and PheWAS catalog (Denny et al., 2010) respectively, and performed additional ICD mapping task to prepare the data for validation analysis. The GWAS evidence of the DiseaseConnect database, which we used to build the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 associations, comes from the National Human Genome Research Institute (NHGRI) GWAS catalog (Welter et al., 2014), whereas for validation, we used the UCSC-GWAS Genome Browser. We have ensured that the GWAS data used to build the networks and to validate them have a very small overlap (
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 8).</br> </br> 2.2 Evaluating the Quality of ICD Disease Taxonomy</br> Here, we systematically evaluated the consistency of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories in ICD taxonomy from both clinical phenotype and molecular profiles (details are in 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 2). We investigated the quality of ICD 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy by evaluating the correlation between the closeness of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs in the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy and the underlying molecular connections (and symptom phenotype similarities) between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs. For example, if two 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs have close positions (e.g. have a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    low level common parent disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
) in the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy, then we would expect that those 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs might have common genes or shared protein-protein interactions or similar phenotypes. We calculated the category similarity between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs using a widely used semantic similarity measure (i.e. Lin measure using information content) (Lin, 1998; Pesquita et al., 2009) to represent the closeness of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs located in the ICD taxonomy. Information theoretic measures such as information content have been used in the context of ICD-9-CM previously (Dahlem et al., 2015). The category similarity measure takes as input two concepts c1 and c2 and outputs a numeric measure of similarity. If two ICD codes have a very specific common parent code in the taxonomic tree structure, then the category similarity would be ~ 1.</br> </br> The molecular and phenotype similarity between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs are calculated by evaluating the shared genes and their GO annotations, molecular network similarities, and shared phenotypes by established similarity measures (e.g. Cosine measure and Jaccard measure). In particular, to propose a more robust representation of molecular network profiles of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, we partitioned the STRING network into 314 topological modules (Data S2) and used them to construct the relevant module vectors of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 using Odds Ratio (OR) as weighting measure. For example, an ICD 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 code would be represented with a 314-dimensional vector, which has a value of wij if its related gene is in a module or 0 otherwise. Suppose we have N genes in total and mi genes of a module i. Now for a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 dj with nj genes, which has kij overlapping genes with the module i, we calculated the value of wij as the following equation, (1) wij=kij/nj−kijmi−kij/N−nj−mi+kij</br> </br> We used the cosine measure to calculate the molecular module similarity between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs after the molecular module vector (i.e. OR weighting) of each 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 was constructed.</br> </br> Furthermore, as ICD taxonomy proposes a framework for organizing the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, it is expected that there should overlapping molecular interactions or phenotype relationships between the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 of the same chapters than those of the different chapters. Thus, we assumed that when we collapse the ICD chapters as the module annotations, such that all the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in one chapter would be considered as members of a same module, the modularity of the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 association networks, i.e. the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 networks with molecular or phenotype associations as links, would reflect the quality of ICD 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy. This means that the higher the modularity, the higher the quality of the ICD chapters as a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 category framework.</br> </br> To evaluate the quality of community structures in complex network, the modularity measure (Newman, 2006) was proposed to quantify the extent to which the connection in communities is above the random expectation in the whole network. Let a network have m edges and Avw be an element of the adjacency matrix of the network. Suppose the vertices in the network are divided into communities such that vertex v belongs to community cv. Then the modularity Q is defined as: (2) Q=12m∑vwAvw−kvkw2mδcvcw where the function δ(i, j) is 1 if i = j and 0 otherwise, and kv is the degree of vertex v. The value of the modularity lies in the range [−1/2,1]. It is positive if the number of edges within groups exceeds the number expected on the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    basis
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 of chance. Otherwise, it would be negative. We use it to measure the consistency of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories (ICD chapter or NCD) as an annotation of topological module (or community) structures within 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 networks. We hypothesize that if a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 category framework captures the molecular or phenotypic profiles of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, then there would be more links existing between the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 members in a category than random expectation.</br> </br> 2.3 Measuring the Disease Specificity</br> As a quantification of the molecular diversity (or the inverse specificity) of a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, we calculated the maximum betweenness of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-related genes in the PPI network (Data S3). Betweenness (Freeman, 1977) is a widely used centrality measure to quantify how many shortest paths run through a given node. In particular, bridging nodes that connect disparate components of the network often have a high betweenness. The betweenness centrality of a node v is given by: (3) bcv=∑s≠v≠tnstvgst where nst(v) denotes the number of shortest paths from s to t that pass through v and gst is the total number of shortest paths from s to t. We will adopt the convention that nstvgst=0 if both nst(v) and gst are zero. We assume the molecular diversity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 would largely lie on the related genes with maximum betweenness. For example, to quantify the molecular diversity (in terms of maximum betweenness) of 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    Alzheimer's disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    AD
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
), we calculated all the betweenness values for the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    AD
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-related genes, such as APP, APOE, TNF and NOS3. Finally, we considered the molecular diversity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    AD
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 as 8.44e-3 since we found that APP has the maximum betweenness of 8.44e-3 among those genes (see Fig. S5a). In fact, this kind of measurement has been successfully used in a previous study (Zhou et al., 2014) to evaluate the diversity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, which indicated that the diversity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 manifestations has a strong positive correlation with the molecular diversity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
. For 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy with good quality, we would expect it to have its 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    lowest level diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (the leaf nodes in the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    tree-structure disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy) with similar molecular diversities.</br> </br> 2.4 Detection of the Significant Disease-chapter Associations</br> We calculated the edge density to quantify the molecular interactions between ICD chapters. To further detect the significant interactions between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in different chapters, we find an approach to obtain the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 that have significant interactions with 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in chapters other than their own. Given a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 di for investigation, we evaluate whether the proportion of interactions (i.e. edge density) of di to the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 set DCk of a chapter Ck is significantly larger than the average proportion of interactions between the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in Ck (Fig. S6). We use binomial test to filter the significant interacting 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-chapter pairs, in which the edge density of the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 to the chapter is significantly higher than the average edge density of the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in the corresponding chapter (details are in 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 4).</br> </br> 2.5 Multi-category Prediction of Diseases</br> The results showing positive correlations between category similarity and molecular similarity, and the high molecular diversity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    many diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 imply that it would be possible to predict the multi-category map for each 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 using its underlying molecular connections. To demonstrate a pilot method for multiple 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 category prediction by integrating molecular module and shared gene similarities, we provided a novel algorithm to generate the possible associated additional 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories for a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    given disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 with the corresponding molecular association scores. (details are in 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 5, Fig. S7). In this algorithm, we integrated the correlation between category similarity and module similarity with significant 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-chapter associations (which are based on the shared gene similarity) to predict the additional chapters for a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    given disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
. We divide the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs in the same chapter to three subsets, which correspond to those pairs with shared root parents, shared second-level intermediate parents and shared third-level intermediate parents, respectively, to help predict to what degree a pair of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 would be located closely in the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy. The principle of the algorithm adheres to the positive correlation between category similarity (or the closeness of position of the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs in ICD 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy) and molecular profile similarity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs, which means that strong molecular profile similarity between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs would indicate close locations of them in the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy. To ensure detecting the significant 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-chapter associations, we next filtered the predicted 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-chapter associations with positive association scores by the significant 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-chapter interactions based on shared genes.</br> </br> 2.6 Construction of Integrated Disease Network</br> To integrate 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 associations derived from both molecular and phenotype features, we performed several sequential analytical steps to generate a highly reliable 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network with strict filtering criterions of the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 links (details are in 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 6). Firstly, we generated three 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 association networks: 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network with module similarity (MSDN) with 598,420 links and 1744 nodes, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network with shared genes (SGDN) with 133,469 links and 1868 nodes, and 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network with symptom similarity (HSDN) with 1,639,791 links and 1814 nodes (Fig. S10 &amp; Table S10) according to shared genes, shared phenotypes and molecular module similarity, respectively. To reduce the possible noise and bias of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 related data sources, we applied a multi-scale backbone algorithm (Serrano et al., 2009) to obtain high reliable 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 links (with significantly high weights than the random expectations) from the three 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 networks. We finally obtained 53,241, 8554 and 134,370 high reliable links for MSDN, SGDN and HSDN, respectively and retained most nodes (1744 for MSDN, 1782 for SGDN and 1814 for HSDN) of these networks. To further reduce the possible weak associations (the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs with high module similarity but no direct protein interactions) derived from module similarity, we calculated the minimum length of the shortest paths (MSPLs) between each 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs and used it as a filtering criterion (with MSPL≤1) for MSDN, which resulted in a more biological meaningful subset of MSDN with 33,611 links and 1694 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
.</br> </br> SGDN would capture strong associations between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs if they have high degree of shared genes even their related genes are not forming functional modules. However, MSDN would give high weights for 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 links if the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs have similar co-locations on the topological modules of molecular network even they have no shared genes. Therefore, MSDN and SGDN are actually two complementary molecular association evidences for 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs and we finally obtained the union of the subset of MSDN and SGDN as the molecular association 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network (MADN), which contains 35,389 links and 1811 nodes with the weights derived from the two original networks. Next, we adopted a highly strict criterion to obtain an integrated 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network (IDN) from the fusion of MADN and HSDN links, which contains 35,114 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 links and 1857 nodes.</br> </br> 2.7 Overlapping Category Detection from Integrated Disease Network</br> Finding the overlapping 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories could be transformed to the task of detecting the overlapping communities (i.e. modules) from the IDN. BigClam (Yang and Leskovec, 2013) is a state-of-the-art overlapping community detection algorithm based on a variant of non-negative matrix factorization, which achieves near linear running time and comparable high quality community results. We used the BigClam algorithm, which is packaged in SNAP complex network software (http://snap.stanford.edu/snap/) to automatically detect overlapping communities from IDN network. Finally, we obtained 223 overlapping 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 communities with 1797 distinct ICD 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 codes. These 223 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 subcategories contain different numbers of ICD codes, ranging from 5 to 168 (Fig. S12 &amp;Data S10).</br> </br> To obtain a top-level category framework of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 corresponding to the chapters in ICD taxonomy, we calculated the overlapping degree of the 223 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 sub-categories by using Jaccard similarity to measure the common number of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 held by two given 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories. This generated a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 category network with 2685 links representing shared ICD codes (a link is established if two 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories share at least an ICD code and the weights of links correspond to the Jaccard similarity) and nodes representing 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories. After that, we clustered the 223 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 sub-categories additionally by a widely used non-overlapping community detection algorithm (considering the link weight and setting the resolution parameter as 0.5) into 17 top-level categories (which corresponds to the number of original chapter-level categories in ICD, which we named as New Chapters, NCs) using the shared ICD codes (Fig. S11c &amp; Data S10). The modularity of these 17 top-level categories (this makes a good comparable partition with ICD chapters) in the network of 223 sub-categories is 0.426, which means a rather good partition of the network. These 17 NCs contain different numbers of sub-categories ranging from 4 to 25 or of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 ranging from 53 to 369 (Fig. S11c &amp; Data S10), covering 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 from all of the 17 chapters of ICD taxonomy (Table S11). These 17 NCs would still contain overlapping 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 codes since the 223 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 sub-categories have overlapping 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 codes. Therefore, 17 NCs with 223 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 sub-categories form a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy consisting of two hierarchical levels with polyhierarchical categories although with a limited number (1797) of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 members.</br> </br> 2.8 Statistical Validation of NCD From External Data</br> To validate the robustness of NCD, we obtained two external phenotype-genotype data sources (i.e. UCSC-GWAS and PheWAS catalog), which have not been integrated yet for generating NCD for further investigation. By measuring whether the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 members in the sub-categories in NCD tend to incorporate the associations of shared genes from these two data sources, we would be able to validate the quality of NCD. If the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in such NCD sub-categories would tend to involve shared genes, then the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 would be more likely associated with one another than 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    other diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
. To test this hypothesis, we obtained the overlapping 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 codes (ODC) in both NCD and the two external phenotype-genotype association databases and evaluate the degree of these 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    ODC disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 links in each NCD sub-category when considering two 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 linked if they share common genes. In detail, we firstly obtained the common 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 codes involved in both NCD and UCSC-GWAS or PheWAS database. Then we generated a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network with shared genes derived from the two datasets, in which two 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 linked if they shared at least one common gene. After that for each NCD sub-category, we generated a complete 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network with the ODC 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in it and overlaid the network on the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network with shared genes. Finally, the overlapping percentage of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 links would be calculated for evaluating the degree of molecular associations involved in 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in each NCD sub-categories (details are in 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 8, Figs. S23–S25).</br> </br> 2.9 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    Statistical Analysis
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
</br> We use R 3.1.0 as the main statistical tool in our work. The comparison of two percentages was calculated by Binomial test or Chi-squared test. Wilcoxon rank sum test was used for compare two independent list of values (e.g. two types of molecular diversities and two groups of MSPLs). All the correlations between two variables were calculated by Pearson's product moment correlation coefficient. Due to the incompleteness and bias of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-related data (i.e. 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-gene associations and 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-symptom associations), we need to distinguish the information from the background noise. Therefore, for comparison with random expectation, we reshuffle (100 random permutations) the symptom features and the related genes of each 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 using the Fisher-Yates method (Fisher and Yates, 1948). The calculations from random permutations were used for the correlation between category similarity and molecular similarity, as well as phenotype similarity. In addition, this was used for detection of the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories with high molecule diversity.</br> </br> 3 Results</br> 3.1 Category Similarity of ICD Taxonomy</br> We curated 1883 distinct ICD 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 codes (Table S1) from the 5-level tree structure of 14,292 ICD-9-CM codes, as well as high confidence protein-protein interactions consisting of 15,551 nodes and 218,409 edges (Franceschini et al., 2013). We compiled 153,277 distinct 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-gene associations between 4552 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    distinct diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in UMLS codes and 14,975 genes reported in the DiseaseConnect database (Liu et al., 2014) (Fig. S3). Next, by manually mapping the DiseaseConnect identifiers to ICD codes, we obtained 160,754 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-gene records involving 1883 distinct ICD codes and 14,906 genes (Figs. S1–2 and Data S1).</br> </br> To evaluate the closeness of two 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in the ICD tree structure, we applied an established semantic similarity algorithm named category similarity (see Methods, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 2.1). This similarity measure is based on the information content, which quantifies the specificity of a term and can be applied to any categorization scheme that has a rooted tree structure, including the ICD-9-CM 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 classification. We then created a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network comprising 1883 nodes (representing ICD codes) and 154,563 edges. The edge weight reflects the category similarity values and higher values reflect higher similarity between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 whose code positions are adjacent in the ICD tree. The category similarity distribution showed that most 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs (135,271, 87.52%) had similarities between 0.2 and 0.5 (Fig. S4a). 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    Disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs within this range mostly belong to different 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 subcategories in the same chapter, such as 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 of other endocrine glands and 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disorders
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 of thyroid gland. For example, type 2 diabetes (ICD: 250.00) and simple goiter (ICD: 240.0), which are in ICD chapter 3, have a category similarity of 0.37. However, there do exist 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs with high category similarities, such as type 2 diabetes (ICD: 250.00) and type 1 diabetes (ICD: 250.01) with a category similarity 0.83. Overall, this measure indicates the capability of ICD in bringing together 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    similar diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in its tree structure, and the overrepresentation of lower similarity scores is indicative of its limitations in doing so.</br> </br> While the ICD classification was derived from clinical manifestations (including symptoms and signs) and does not necessarily reflect the connections among the molecular components of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, it is informative to quantify to what extent it carries molecular information. We investigated the correlations of category similarity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs with 1) the degree of shared genes and shared clinical phenotypes, 2) GO term (Cell Component, Molecular Function, Biology Process) similarity (Mistry and Pavlidis, 2008), and 3) topological similarity (i.e., minimum shortest path length and molecular module similarity) among them (Methods, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 2.2).</br> </br> We found that close 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 codes (
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs with a high category similarity) actually have higher clinical phenotype similarity (Methods, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 2.3), which adheres to the construction principle of ICD taxonomy based on symptom phenotypes (Fig. S4b, PCC = 0.960, 95% CI = [0.854, 1.000], p = 2.079e-05). Furthermore, we observed strong correlations between higher category similarity bins for molecular profiles, compared to lower category similarity bins (Fig. S4c–i and Table S2. See Methods, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 2 for detailed information). In particular, we observed that in addition to the strongly positive correlations, the percentage overlap of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs with shared genes was generally larger than the random controls (Fig. S4c and d, see Methods, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 2.4). The top 10 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs with the largest number of shared genes are all from Chapter 2, which consists of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    cancer
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 types. This might reflect the fact that 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    cancers
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 are the most studied and complex 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 phenotypes involving various gene mutations (Table S3, see Methods, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 2 for detailed information).</br> </br> Overall, these findings indicate that 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in the same ICD chapter tend to have a higher degree of shared genes, and the closer their positions in the ICD tree, the higher is the degree of shared genes.</br> </br> 3.2 Heterogeneity of Molecular Specificity in ICD Taxonomy</br> We measured the maximum betweenness of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-related genes in the protein-protein interaction (PPI) network to quantify the molecular diversity (the inverse of specificity) of each 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, as described previously (Zhou et al., 2014) (see Methods, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 3). A high maximum betweenness indicates a high molecular diversity. For example, the molecular diversity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    Alzheimer's disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 could be represented by the maximum betweenness of its related genes (i.e., the betweenness of the APP gene) in the PPI network (Fig. S5a).Fig. 2 Lack of molecular specificity in ICD taxonomy and the blurred boundary between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories in ICD taxonomy. a. The distribution of molecular diversity of 1883 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    ICD diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
; b. The boxplot of molecular diversity of 17 ICD chapters (ordered by median values); c. The 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network with shared genes in which the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 belong to Chapter 5 and Chapter 8. The ICD codes 295, 296, in Chapter 5 have dense relationships to the ICD codes in Chapter 8; d. The 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 category network with shared genes. The nodes indicate the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 chapters and the weights of edges represent the edge densities between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 chapter pairs; the nodes with same color are considered as a chapter cluster, which is detected by community detection algorithm; e. Modularity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 networks with chapter as module annotations; f. The correlation between category similarity and phenotype similarity of ICD chapters.</br> </br> Fig. 2</br> </br> We observed that the molecular diversity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in the ICD taxonomy is 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    heterogeneous
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, with molecular diversity values varying from 10−8 to 10−2 with a median value of 8.93e-04 (Fig. 2a and Data S3). The top two 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 chapters with the highest median molecular diversity were Chapter 2 (3.87e-03) and Chapter 1 (1.31e-03) (Fig. 2b). Furthermore, we found that 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    neoplasm
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (Chapter 2) and 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    infectious disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (Chapter 1) categories tended to have higher molecular diversity compared to their complementary categories (Neoplasms vs. Non-Neoplasms p &lt; 2.2e-16, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    Infectious diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 vs. 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    non-infectious diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 p = 2.0e-02, Fig. S5b–c) and random controls. We also found that 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories annotated as “other/unspecified” categories (
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 3.1) had higher molecular diversity compared to 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories with specific conditions (p = 9.75e-03, Fig. S5d, Data S4; see 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 3.1) and its random control. These results indicate that the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    neoplasms
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
, 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    infectious diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
, and “
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    Other/unspecified diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
” categories should be further investigated for molecular subtypes. A detailed discussion of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 cases is offered in 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 3, Data S5 &amp; Tables S4–S5.</br> </br> 3.3 The Blurred Molecular Boundary Between ICD Categories</br> In the current ICD taxonomy, we observed many instances where there exists a significant number of links between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in different chapters, comparable to the number of links between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 within the same chapter (Table S6 &amp; Fig. 2d, see Methods &amp; SM Section 4). For example, strong shared-gene relationships were detected between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    respiratory diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (Chapter 8) and 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    mental, behavioral, and neurodevelopmental disorders
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (Chapter 5) (Fig. 2c–d, more examples shown in 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 4, Tables S7–9). In addition, by calculating the shared molecular connections between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in the context of chapters, we could detect 768 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 with a significant number of shared genes with 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases other
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 than those in their own chapters (Data S6 &amp; SM Section 4).</br> </br> To further quantify the molecular boundaries between the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories in ICD 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy, we evaluated the modularity, a structural measure of the tendency of the network to form close-knit communities (see Methods, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 2.5), generated by either shared molecular profiles or shared phenotypes. When we mapped ICD chapters as grouping annotations on the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 networks filtered by with appropriate weight thresholds, and calculated their modularity, we observed very low modularity values (Fig. 2e). Since modularity is a widely used measure to validate the quality of partitions/module structures in complex networks, this means that the grouping of ICD chapters does not agree with the natural topological groupings of their corresponding molecular networks (
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 modules). This finding gives strong evidence for the blurred 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 boundaries of the ICD taxonomy, possibly arising from the complexity of the underlying molecular mechanisms, in particular the possible overlap of their respective subnetworks, or 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 modules, in the interactome.</br> </br> Furthermore, although the modularity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 networks with shared phenotypes (similarity ≥ 0.1) is slightly positive, the weak correlation (PCC = 0.08, p-value = .7588) between phenotypic similarity and category similarity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs in each chapter (Fig. 2f) indicates that ICD taxonomy does not adequately incorporate phenotype similarity knowledge into 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 category structures. These observations indicate that the strict tree structures in the ICD taxonomy wherein terms can only have one lineage (Cimino, 2011) may be inefficient for 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 classification given the contemporary knowledge of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pathobiology, and should therefore be refined to be polyhierarchical in structure.</br> </br> 3.4 Polyhierarchical Mapping of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    Diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Using Molecular Module Similarity</br> It has been proposed that if two 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 modules overlap in the molecular interaction network, local perturbations in one 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 might disrupt the biological pathways in the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    other disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, resulting in shared pathobiological characteristics (Menche et al., 2015). We observed a strong positive correlation between category similarity and module similarity (see Methods, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 5.1) of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, indicating that two 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 with higher module similarity would be more closely localized in the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 category (Fig. 3a, PCC = 0.887, 95% CI = [0.584,0.973], p = 6.12e-04; 3b, PCC = 0.974, 95% CI = [0.889,0.994], p = 2.08e-06).Fig. 3 Polyhierarchical map prediction of ICD taxonomy based on molecular module similarity. a. Correlation between mean semantic (category) similarity and mean modular similarity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs; b. Correlation between overlapping edge ratio with category similarity and modular similarity of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs; c. Correlation between predicted category number and molecular diversity of ICD codes in Chapter 14 (PCC: 0.438; 95% CI: [0.401, 0.474]; p &lt; 2.2e-16); d. Polyhierarchical map of the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 codes in Chapter 8, indicating that the 20 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 codes in Chapter 8 have significant associations with two 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 category clusters: 1) Chapter 1 (
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    infectious disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
) and Chapter 2 (
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    neoplasms
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
); 2) Chapter 3 (endocrine, nutritional and 
<mark class="entity" style="background: #FF0000; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    metabolic diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FN</span>
</mark>
 and 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    immunity disorders
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
), Chapter 5 (
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    mental disorders
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
), Chapter 6 (
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    nervous diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
), Chapter 9 (
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    digestive diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
), Chapter 12 (skin and 
<mark class="entity" style="background: #FF0000; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    subcutaneous disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FN</span>
</mark>
) and Chapter 13 (musculoskeletal system and 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    connective tissue diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
); e. The boxplots of phenotype similarity of predicted 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs and original ICD 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs in the same top-level chapters (p &lt; 2.2e-16, Wilcoxon test).</br> </br> Fig. 3</br> </br> Here, we utilize the module similarity between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs to predict the categories of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    similar diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
. In particular, we determine the taxonomic closeness (
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 2.1) of each given 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pair to predict the additional categories of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, by applying heuristic rules incorporating the positive correlation between category similarity and module similarity (see Methods, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 5.1). In particular, using the 598,420 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs with positive module similarity (Data S7), we generated 2057 predicted additional category results for 722 out of 1883 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 codes (38.3%) in which each 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 code had ~4 categories on average (Data S7&amp;8). We found that the number of predicted categories positively correlated with the molecular diversity of the original 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 codes (Fig. 3c, PCC = 0.547, 95% CI = [0.514, 0.578], p &lt; 4.94e-324; for External validations see 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 5.2). This indicates that 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 with multiple pathogenic pathways could be captured by polyhierarchical mapping. For example, the 20 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in Chapter 8 (i.e. 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    Diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 of the Respiratory System) have been predicted to belong to over five additional chapters, such as 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    neoplasms
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
, 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    infectious diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
, and 
<mark class="entity" style="background: #FF0000; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases of the skin
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FN</span>
</mark>
 and 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    subcutaneous
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 tissue (Fig. 3d), which is consistent with the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    heterogeneous pathogenesis
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    COPD
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 and 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    asthma
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (Grainge et al., 2016; Sharma et al., 2015). A detailed discussion on the polyhierarchial map of the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    mental disorders
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 is offered in 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 5.3 (Fig. S8).</br> </br> Furthermore, we found that the predicted category framework, which is based only on molecular module similarity, also had higher phenotype similarity than 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 with shared root codes in the original ICD chapters (see 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 5.2, median: 0.0703 vs. 0.0563; mean: 0.125 vs 0.109; p &lt; 2.2e-16, Fig. 3e). This observation helps to establish that the predicted category results are of higher quality than ICD with respect to their phenotype homogeneity.</br> </br> 3.5 Integrated Disease Network for Overlapping Disease Classification</br> To extend and redefine 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 concepts by discovering additional categories of a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, we generated a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    novel disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy by constructing an integrated 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network (IDN) with: (a) Shared clinical phenotypes including shared symptoms; (b) Shared molecular profiles including (i) shared genes and molecular module similarity and (ii) shortest path lengths in the PPI network, based on a systematic integration process to filter out possible false positive associations (see Methods, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 6, Fig. S9 and Fig. S11a), which includes 1857 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 and 35,114 links (Data S9).</br> </br> Next, we applied high performance community detection algorithms to identify overlapping community structures in the IDN (see Methods, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 7 and Fig. S11a). In particular, we first used BigClam (see Methods) since this method is able to detect overlapping communities whereby a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 can belong to multiple communities, in line with our main premise of creating a molecular based 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    flexible disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 classification. This resulted in 223 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 sub-categories with 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    overlapping diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 as members (Fig. S11a and Data S10), which included 1797 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    distinct diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 from the ICD taxonomy. These 223 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 sub-categories contain different numbers of ICD codes, ranging from 5 to 168 (Fig. S12), therefore, they represent different levels of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories similar to ICD chapters and their sub-categories.</br> </br> To further develop a more unified view of the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 category quality, we used the established BGLL method (Blondel et al., 2008), which detects non-overlapping communities, to cluster these 223 sub-categories further into 17 non-overlapping, distinct parts, such that these represent the 17 new chapter-level categories (called new chapters, or NCs) using the shared ICD codes (see Methods, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 7.2, Fig. S11b). Overall, this clustering order effectively ensures distinct top-level categories that have overlapping subcategories.</br> </br> The resulting 17 NCs contain different numbers of sub-categories ranging from 4 to 25, or of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 ranging from 53 to 369 (Fig. S11c). We denote the 17 NCs together with their 223 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 sub-categories as our new overlapping 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 classification (NCD). Each of the resulting NCs reflects the shared features of integrative molecular and phenotypic profiles (SM Section 7.4; Fig. S11c &amp; Table S12, Data S11–14). For example, NC08 could be denoted as the “
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    limbic system development-vision disorders-related diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
” since the most enriched PPI module (p = 4.9e-324, Relevance ratio = 0.7778) of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    its constituent diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 was mainly related to biological process; “limbic system development” (p = 1.13e-04), and 73.84% (127/172) of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in NC08 shared the phenotype, “
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    vision disorders
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
” (p = 4.9e-324) (Tables S13–S14).</br> </br> 3.6 New Disease Categories Define 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    Diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 with Clearer Boundaries and Balanced Diversity</br> To confirm the phenotypic and molecular cohesiveness of our overlapping 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories, we compared the modularity of NCD with that of the ICD taxonomy. We found that the 17 NCs consistently have much higher modularity than the original ICD chapters for all types of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 association networks (SM Section 7.3; Fig. 4a–h, Fig. S13). This finding indicates that the phenotypic and molecular links between the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 of a category in NCD are much denser compared to ICD taxonomy.Fig. 4 Properties of new 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories (NCD) and comparison to conventional ICD classification. a. Modularity of phenotype (symptom)-based 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network (NCD vs. ICD); b. Modularity of gene-based 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network (NCD vs. ICD); c. Modularity of molecule module-based 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network (NCD vs. ICD); d. Modularity of gene ontology (molecular function)-based 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network (NCD vs. ICD); e. Modularity of gene ontology (biological process)-based 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network (NCD vs. ICD); f. Modularity of gene ontology (cellular component)-based 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network (NCD vs. ICD); g. Modularity of shortest path-based 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network (NCD vs. ICD); h. Modularity of GWAS shared gene-based 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network (NCD vs. ICD); i. Percentage of minimum shortest path lengths within the range [0, 2] of intra-category and inter-category (NCD vs. ICD, chi-squared test); j. The number of overlapping 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs from NCD with shared genes from GWAS, compared to random expectation, binomial test; k. The number of overlapping 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs from NCD with shared genes from PheWAS, compared to random expectation, binomial test; l. The number of predicted NCD categories for ICD codes (i.e. 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
) as a function of their molecular diversity.</br> </br> Fig. 4</br> </br> Furthermore, to ensure that the performance of NCD is indeed due to the combined effect of the molecular and phenotypic profiles, we performed a controlled experiment where we determined the new 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories based on molecular-based networks and phenotype-based networks only by running the entire analytical pipeline and applying the same category prediction algorithm (
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 7.3). We found that NCD outperforms both molecular-based categories and phenotype-based categories in capturing the gene similarity, GO term similarity and phenotypic similarity (Figs. S14–S16). This suggests the importance of integrating both clinical phenotypes and molecular profiles to obtain a high-quality 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy.</br> </br> Furthermore, we found that the minimum shortest path lengths in the PPI network between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs that belong to the same NCD categories had a larger percentage of low values (i.e., [0,2]) compared to ICD (Fig. 4i, 62.86% vs 58.95%, p &lt; 4.9e-324; SM Section 7.3). This result indicates that 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 within an NCD category have a significantly higher degree of shared genes (or shorter path lengths) in comparison to 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 within a category in ICD. On the other hand, the MSPLs between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs in different NCD categories had a significantly lower percentage of low values than those in the ICD (47.27% vs 54.88%, p &lt; 4.9e-324, Fig. 4i &amp; Fig. S17, External validations in SM 8.3), which indicates a lower degree of shared genes (or shorter path lengths) between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 from different categories in NCD than in ICD. These findings demonstrate that our NCD framework has clearer boundaries between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    distinct diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 belonging to different categories than those in the original 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    ICD disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy.</br> </br> Moreover, to validate the robustness of NCD predictions, we calculated the degree of associations in terms of network density among the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in each sub-category of NCD. To this end, we investigated the overlaps with the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 pairs connected by shared genes using two independent phenotype-genotype association databases, GWAS and PheWAS (see Methods, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 1.2,1.6 &amp; 8). We found that for the 223 sub-categories in NCD, network density was significantly higher compared to random controls (GWAS: p-value = 9.42e-197, Fig. 4j; PheWAS:p-value = 1.31e-14, Fig. 4k). This means that the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in the 223 sub-categories in NCD would tend to have shared genes. For example, the New Chapter: NC12 in NCD, including 11 sub-categories and 136 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    ICD diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (belonging to eight ICD chapters), is enriched with 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    respiratory and airway diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (e.g. 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    COPD
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 and 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    asthma
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
).</br> </br> We obtained 37 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    overlapped diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 from the GWAS database, which have a high degree of shared genes with the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in each sub-category of the NC12 (Fig. 5a). In particular, the sub-categories, such as NC12.M06 (p-value = 2.53e-30), NC12.M03 (p-value = 1.80e-38) and NC12.M02 (p-value = 6.89e-19) have significantly higher density than those of the whole GWAS 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network (Fig. S22). Furthermore, we found that the overlapping subcategories of the NCD are able to differentiate between different components (i.e. 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    asthma
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
/allergy vs. 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    COPD
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
) of the same broad group of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (i.e. 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    respiratory diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
) (see Fig. S22 for a detailed example). Indeed, in the NC12 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 chapter chiefly containing 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    respiratory diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
, the two sub-categories, namely NC12.M06 and NC12.M07, overlap in the underlying molecular interaction network while still containing the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    respective disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    asthma
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 and 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    COPD
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, respectively) genes separately (Fig. 5a). A detailed discussion is offered in 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 8(with results in Data S21–22, Tables S18–19 &amp; Figs. S23–S25).</br> </br> In addition, in NCD, a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 can be classified into multiple categories, and the number of categories of a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 positively correlates with its molecular diversity (Fig. 4l, PCC = 0.352, 95% CI = [0.311, 0.392], p-value &lt; 4.94e-324; External validations in SM 8.3). For example, we reclassified 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    neoplastic diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 into multiple categories due to their high molecular diversity. Two hundred and fifty-eight 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    neoplastic diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 in our NCD were divided into 144 sub-categories and 17 NCs (Figs. S20 &amp; S21). Thirty-nine out of 144 sub-categories (27.08%) were enriched with 
<mark class="entity" style="background: #FF0000; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    “neoplasm” diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FN</span>
</mark>
 (Data S19, p-value = 2.78e-5). There were mainly 4 NCs (i.e., NC01, NC06, NC11, and NC16) containing these 32 sub-categories and 188 
<mark class="entity" style="background: #FF0000; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    “neoplasm” disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FN</span>
</mark>
 codes (Fig. 5b), where 76.06% (143/188) of the 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    neoplastic diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 were classified into &gt;1 sub-category, ranging from 2 to 15(Data S20 &amp; Table S17). The 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    neoplasm
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 with the highest molecular diversity, “
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    malignant neoplasm
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 of connective and other soft tissue” (ICD: 171; molecular diversity: 0.035), was reclassified into 15 sub-categories, and “
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    malignant neoplasms
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 of thyroid gland” (ICD: 193; molecular diversity: 0.0028) was assigned to 14 sub-categories. Furthermore, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    related diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 had been reclassified together in NCD, such as the well-known 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-correlations among 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    H. pylori infection
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (ICD: 041.86), 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    stomach cancer
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (ICD: 151), and 
<mark class="entity" style="background: #FF0000; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    duodenal ulcer
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FN</span>
</mark>
 (ICD: 532) or 
<mark class="entity" style="background: #FF0000; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    peptic ulcer
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FN</span>
</mark>
 (ICD: 533) (Fig. 5b) (Sitas, 2016; Graham, 2015).</br> </br> More interestingly, some 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, like 
<mark class="entity" style="background: #FF0000; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    viral hepatitis C
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FN</span>
</mark>
 (ICDs: 070.4, 070.5, 070.7), 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    graft-versus-host disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (ICDs: 279.5/279.50), 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    glomerulonephritis
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (ICDs: 580, 582, 582.9), 
<mark class="entity" style="background: #FF0000; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    circumscribed scleroderma
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FN</span>
</mark>
 (ICD: 701.0), 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    systemic lupus erythematosus
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (ICD: 710.0), and 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    rheumatoid arthritis
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (ICDs: 714/714.0), each from different chapters in ICD taxonomy, were classified together into a unique NCD sub-category (NC06.M10) since 50% (13/26) of these 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 share a PPI module related to immune response (
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 7.4, Fig. 5c, Data S15–16, Tables S15–16).</br> </br> In addition, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 originally in the same ICD chapter, such as 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    viral pneumonia
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (ICD: 480) and 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    influenza
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (ICD: 487) from respiratory 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    system-related diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (Chapter 8), were reclassified into different categories in the NCD (NC12, NC10). Influenza shared more phenotype profiles with “
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    episodic mood disorders
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
” (ICD: 296) in NC10.M01, rather than 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    viral pneumonia
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 in NC12 (Fig. S18&amp; Data S17), which is in accordance with recent epidemiological studies between 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    episodic mood disorders
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 and 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    influenza
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (Okusaga et al., 2011; Canetta et al., 2014), and, furthermore, we also found that 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    influenza
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 shared some molecular profiles with “
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    episodic mood disorders
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
” (ICD: 296) in NC10.M01 (Fig. S19, Data S18).</br> </br> These findings suggest that NCD offers a promising integrative framework incorporating both clinical phenotypes and molecular profiles for 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy that has very practical implications for the precise investigation of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 subtyping and etiologies.Fig. 5 Biological insights of new 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy. a. The New Chapter containing 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    airway diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (NC12) consists of 11 sub-categories and 136 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    ICD diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 belonging to 8 ICD chapters. The subcategories overlap in the underlying molecular interaction network, while still separately including the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 genes (
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    asthma
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 and 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    COPD
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
, respectively) that characterize each subcategory; b. The 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network of 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    neoplasms
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 in NCD. The 32 sub-categories significantly representing 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    neoplasms
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 are divided into 4 NCs (G1). 
<mark class="entity" style="background: #FF0000; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    Helicobacter pylori
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FN</span>
</mark>
 [
<mark class="entity" style="background: #FF0000; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    H. pylori
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FN</span>
</mark>
] (041.86), 
<mark class="entity" style="background: #FF0000; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    malignant neoplasm of stomach
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FN</span>
</mark>
 (151), 
<mark class="entity" style="background: #FF0000; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    duodenal ulcer
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FN</span>
</mark>
 (532), 
<mark class="entity" style="background: #FF0000; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    peptic ulcer
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FN</span>
</mark>
, site unspecified (533), which have significant relationships, are clearly clustered into a sub-category (NC11. M07) (G2); c. A sub-category (NC06.M10) in NCD, which includes 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 from 8 different ICD chapters with shared molecular mechanism and phenotypes. Fifty percent (13/26 = 50%) of the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 in NC06.M10 share a PPI module, the biological function of which is enriched with immune system response, while over 90% (25/26 = 96.2%) of the shared common phenotype of this module is “Pain”.</br> </br> Fig. 5</br> </br> 4 Discussion</br> Given the molecular network mechanisms (Barabasi et al., 2011; Zanzoni et al., 2009), genetic pleiotropy (Solovieff et al., 2013), as well as complicated genotype-phenotype associations 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    underlying diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, the establishment of a molecular-based 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy with clear boundaries is essential but challenging. From the molecular network perspective, we first investigated the utility, shortcomings, and inconsistencies of ICD-9-CM, the established 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy for clinical settings. We found that there exist a considerable number (~40% of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    our investigated diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
) of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, for example, 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    cancer
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 and 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    infectious diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
, that have diverse molecular network mechanisms and tend to interact more with 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 from other chapters. It is also these molecularly 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diverse diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 that mainly contribute to the blurred boundary of ICD 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy (see Methods, 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    SM
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 Section 4&amp;7). Upon exploring the molecular diversity and cross-chapter interactions between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, we propose a novel 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 classification system based on the integration of the clinical and molecular profiles of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
. In particular, we integrate 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 networks taking into account molecular and phenotypic connectivity among 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
, predict the multiple 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories that 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 belong to, and finally validate the biological cohesiveness of our NCD by network topological measures such as modularity and shortest path length.</br> </br> Our findings indicate that although general correlations exist between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 closeness in ICD taxonomy and underlying molecular profiles, ICD still displays significant limitations with regard to the heterogeneity of molecular diversity and clear category boundaries. In our NCD, a 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 with a high molecular diversity tends to be classified into multiple 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories, which indicates that there exist more 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 subtypes for that 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
. For example, “
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    malignant neoplasm
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 of the pancreas” was reclassified into 11 sub-categories and 4 NCs, which is consistent with a recent study wherein 4 phenotypic subtypes of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    pancreatic cancer
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 were enriched for 10 distinct molecular mechanisms (Bailey et al., 2016). Therefore, we believe that the new 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 classification system may help facilitate precise clinical diagnosis and correct prognosis (Jameson and Longo, 2015), and does so in alignment with refined molecular network diagnostics.</br> </br> Furthermore, the molecular network underpinnings and overlapping 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories of NCD provide a credible relationship map between 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 and 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories that may radically transform our current understanding of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 and relevant treatment paradigms. On the one hand, our approach accurately links 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 with all possible underlying mechanisms in the molecular interaction network. On the other hand, it presents a promising approach to the identification of targeted drugs for the treatment of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    related diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
. For example, 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    breast cancer
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 and 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    influenza
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
 (both in NC11.M02) may share potential drug targets (Park, 2012). As another example, metformin, widely prescribed to treat 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    metabolic syndrome
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (in NC11.M02), could alter the gut microbiome composition and function, improve gut 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    microbial dysbiosis
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (Forslund et al., 2015; Cabreiro et al., 2013), and also prevent 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    colorectal cancer
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (also in NC11.M02) through microbiome-influenced immune response modification (Nakatsu et al., 2015). Here, it is important to note that while a considerable number of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 have a strong environmental component, our main focus has been the many diverse molecular determinants. In the future, additional environmental factors such as epigenetic changes can be added into the data integration scheme to further refine the classification.</br> </br> There exist several potential limitations of this work. Although we have aimed to address the possible confounders by constructing random controls and using external evaluations, the incompleteness and bias incorporated in the integrated data sources are likely to influence the generalization of our results. For example, DiseaseConnect yields an incomplete 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-gene database: 1883 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    ICD diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 could be mapped (Table S11), leading to only 1797 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 included in the NCD. Furthermore, as with other studies that rely on literature-based and ontological knowledge, investigation bias remains an issue, where the molecular mechanisms (e.g. related genes and their interactions) of some 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (e.g. 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    cancer
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
) being more intensively studied than others may influence the results (Menche et al., 2015). We expect the results of similar works to be more refined in the future as biomedical datasets become more complete. The incorporation of more comprehensive 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
-gene data sources, such as 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    DISEASES
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 (Pletscher-Frankild et al., 2015) and MalaCards (Rappaport et al., 2017), could result in an improved study. While we have chosen to keep individual external gene expression datasets outside the scope of this study since gene expression is highly tissue- and cell-type dependent, it presents an interesting future direction and could potentially improve the quality of the resulting 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories if exhaustive lists of tissue-specific expression datasets are used in dedicated studies. In addition, our NCD merely delivers a two-level taxonomy framework without elaborated hierarchical structures in the same 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 categories, which could be further refined or optimized through methods like hierarchical clustering algorithms (Murtagh and Contreras, 2012) and systematic posteriori ontology engineering method (Gessler et al., 2013). Finally, high-quality ontologies, such as the Human Phenotype Ontology (Kohler et al., 2014) and Disease Ontology (Kibbe et al., 2015), can be used for external validation, or for further integration to obtain more robust and extensive NCDs.</br> </br> In this big-data era, the dramatically increasing multi-omics databases, as well as clinical data from electronic health records (EHR) involving phenotypic, therapeutic and environmental factors information (Jensen et al., 2012), should also be incorporated into the new 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy refinement for patient stratification and 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 treatment. At this point, a realistic assumption is that the translation of this classification to the clinic will need some time. That said, while the ICD is originally made “by clinicians for clinicians”, it is now widely used by biomedical researchers as well to gain a deeper understanding of 
<mark class="entity" style="background: #00FF00; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    human diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">TP</span>
</mark>
. We therefore believe that researchers will be the first and direct beneficiaries of our approach.</br> </br> In conclusion, our study provides valuable insights into the polyhierarchical network-based 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 classification beyond the traditional tree structure. Our 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    integrated disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 network approach is sufficiently powerful to elucidate the tangled underpinnings of 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    human diseases
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 and uncover distinct 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 boundaries. Our work may provide a new framework for the 
<mark class="entity" style="background: #FF00FF; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
    disease
    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">FP</span>
</mark>
 taxonomy reform based on big-data fusion, so as to generate further the robust infrastructure needed for precision medicine.</br></div>
</figure>
</body>
</html>